readme.md

```diff
@@ -121,7 +121,8 @@ Config file should be stored at `~/.config/tg/conf.py`. This is simple python fi
 ### Simple config:
 
 ```python
-PHONE = "[your phone number]"
+# should start with + (plus) and contain country code
+PHONE = "[phone number in international format]"
```

Comment:
maybe let's add reexp to validate bad phone number?

Reply:
That's an option but I don't want to add unnecessary logic. Tdlib will tell if phone number is incorrect don't think we need to validate it as well